scripts/inject.js

```diff
@@ -7,7 +7,7 @@ const buildPackage = require("./package");
 
 const useBdRelease = args[2] && args[2].toLowerCase() === "release";
 const releaseInput = useBdRelease ? args[3] && args[3].toLowerCase() : args[2] && args[2].toLowerCase();
-const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "ptb" ? "Discord PTB" : "Discord";
+const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "dev" ? "Discord Development" : releaseInput === "ptb" ? "Discord PTB" : "Discord";
```

Comment:
```suggestion
const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "dev" ? "Discord Development" : releaseInput === "canary" ? "Discord PTB" : "Discord";
```
Discord dev is a clone of canary.

Reply:
Dev is a clone of canary, I do not understand the Suggested change though because ptb is in a separate file location.